beacon-chain/core/state/transition_test.go

```diff
@@ -541,55 +545,3 @@ func TestProcessEpoch_CantGetCurrentValidatorIndices(t *testing.T) {
 		t.Errorf("Expected: %s, received: %v", wanted, err)
 	}
 }
-
-func TestProcessEpoch_CantProcessCurrentBoundaryAttestations(t *testing.T) {
```

Comment:
without genesis slot being 0, these tests are unreachable. they were tested for underflow conditions